easybuild/easyconfigs/s/SunPy/SunPy-3.1.3-foss-2021b.eb

```diff
@@ -0,0 +1,96 @@
+description = """The community-developed, free and open-source solar data analysis environment for Python."""
+
+toolchain = {'name': 'foss', 'version': '2021b'}
+
+dependencies = [
+    ('Python', '3.9.6'),
+    ('astropy', '4.3.1'),
+    ('lxml', '4.6.3'),
+    ('matplotlib', '3.4.3'),
+    ('OpenJPEG', '2.4.0'),
+    ('scikit-image', '0.19.1'),
+    ('SciPy-bundle', '2021.10'),
+    ('tqdm', '4.62.3'),
+]
+
+use_pip = True
+
+exts_list = [
+    ('semantic_version', '2.8.5', {
+        'checksums': ['d2cb2de0558762934679b9a104e82eca7af448c9f4974d1f3eeccff651df8a54'],
+    }),
+    ('jmespath', '0.10.0', {
+        'checksums': ['b85d0567b8666149a93172712e68920734333c0ce7e89b78b3e987f71e5ed4f9'],
+    }),
+    ('asdf', '2.8.3', {
+        'checksums': ['de0f70ffb2e0d539461940d6f7529c3548541fa098d8edc37af256af61c09b44'],
+    }),
+    ('drms', '0.6.2', {
+        'checksums': ['21df2b3caf2aabbd601e7e432a7122ecba74f351856dc16d194faff3d565b7da'],
+    }),
+    ('appdirs', '1.4.4', {
+        'checksums': ['7d5d0167b2b1ba821647616af46a749d1c653740dd0d2415100fe26e27afdf41'],
+    }),
+    ('cached-property', '1.5.2', {
+        'checksums': ['9fa5755838eecbb2d234c3aa390bd80fbd3ac6b6869109bfc1b499f7bd89a130'],
+    }),
+    ('isodate', '0.6.1', {
+        'checksums': ['48c5881de7e8b0a0d648cb024c8062dc84e7b840ed81e864c7614fd3c127bde9'],
+    }),
+    ('requests-toolbelt', '0.9.1', {
+        'checksums': ['968089d4584ad4ad7c171454f0a5c6dac23971e9472521ea3b6d49d610aa6fc0'],
+    }),
+    ('requests-file', '1.5.1', {
+        'checksums': ['07d74208d3389d01c38ab89ef403af0cfec63957d53a0081d8eca738d0247d8e'],
+    }),
+    ('zeep', '4.1.0', {
+        'checksums': ['5867f2eadd6b028d9751f4155af590d3aaf9280e3a0ed5e15a53343921c956e5'],
+    }),
+    ('Glymur', '0.9.7.post1', {
+        'checksums': ['1ad6c113fb1a4392525dbd25a22526abaea05217b1476ea3f33e3b74b79c81e8'],
+    }),
+    ('greenlet', '1.1.2', {
+        'checksums': ['e30f5ea4ae2346e62cedde8794a56858a67b878dd79f7df76a0767e356b1744a'],
+    }),
+    ('SQLAlchemy', '1.4.31', {
+        'checksums': ['582b59d1e5780a447aada22b461e50b404a9dc05768da1d87368ad8190468418'],
+    }),
+    ('async-timeout', '4.0.2', {
+        'checksums': ['2163e1640ddb52b7a8c80d0a67a08587e5d245cc9c553a74a847056bc2976b15'],
+    }),
+    ('multidict', '6.0.2', {
+        'checksums': ['5ff3bd75f38e4c43f1f470f2df7a4d430b821c4ce22be384e1459cb57d6bb013'],
+    }),
+    ('yarl', '1.7.2', {
+        'checksums': ['45399b46d60c253327a460e99856752009fcee5f5d3c80b2f7c0cae1c38d56dd'],
+    }),
+    ('aioftp', '0.20.0', {
+        'checksums': ['37caa22ac58f6854ffb79a757921d2d01c9da17bf800beb2f203f8cf83fd7ec1'],
+    }),
+    ('aiosignal', '1.2.0', {
+        'checksums': ['78ed67db6c7b7ced4f98e495e572106d5c432a93e1ddd1bf475e1dc05f5b7df2'],
+    }),
+    ('frozenlist', '1.3.0', {
+        'checksums': ['ce6f2ba0edb7b0c1d8976565298ad2deba6f8064d2bebb6ffce2ca896eb35b0b'],
+    }),
+    ('aiohttp', '3.8.1', {
```

Comment:
We should make this depend on `aiohttp` from #14784

Reply:
Done in https://github.com/easybuilders/easybuild-easyconfigs/pull/14898/commits/147c06583682388c7916b27d7d505f9f8aa2940f

Reply:
Thanks! :+1: